hal/armv7m/imxrt/10xx/hal.c

```diff
@@ -25,12 +24,13 @@ typedef struct {
 
 struct{
 	intr_handler_t irqs[SIZE_INTERRUPTS];
+	syspage_hal_t *halSyspage;
```

Comment:
Structure name should be `hal_syspage_t`. The structure definition should be provided by HAL for specific platform.